spec/acceptance/ini_subsetting_spec.rb

```diff
@@ -44,18 +42,20 @@
   end
 
   describe 'ensure, section, setting, subsetting, & value parameters => present with subsections' do
+    basedir = setup_test_directory
+
     pp = <<-EOS
```

Comment:
Use a `let`.

Reply:
As it is now, we can't declare basedir variable with let, because it can't be used inside describe/context blocks or to pass it to shared_example block.
I've added at the beginning of the tests: basedir = setup_test_directory and 
removed the duplicates